Resources/Prototypes/Entities/Objects/Consumable/drinks_cans.yml

```
@@ -157,7 +157,7 @@
   parent: DrinkCanBaseFull
   id: DrinkEnergyDrink
   name: Energy drink
-  description: ''
+  description: Gives you wings to do nothing.
```

Not sure what you're going for here?

```suggestion
  description: Gives you wings that do nothing.
```
or something more like
```suggestion
  description: Legally we're not allowed to say this gives you wings.
```